src/components/date-picker/date.types.ts

```diff
@@ -8,3 +8,5 @@ export type DateType =
     | 'year';
 
 export type Languages = 'da' | 'en' | 'fi' | 'nb' | 'no' | 'nl' | 'sv';
+
+export type CustomDateFormatter = (date: Date) => string;
```

Comment:
Do we need to prefix it with `Custom`? I do not think it adds much :slightly_smiling_face:

Reply:
We already have a `DateFormatter` class, but I agree, that shouldn't really determine how we choose to name interfaces that we expose

Reply:
I just inlined this type, and it's only used in two places now.

Reply:
Yeah, I wanted to name it `DateFormatter` but I couldn't 🤣 
Inlining it is fine I guess.